src/components/CookieWarning.js

```diff
@@ -47,13 +46,13 @@ const CookieWarning = () => {
         and to improve your experience. By clicking Agree or continuing to use our site, you consent to the use of these functional
         cookies. If you do not wish to allow use of these cookies, you may tell us that by clicking on Reject. As a result, you will be unable
         to use our site. To find out more, read our `,
-        h(Link, { style: { textDecoration: 'underline' }, href: Nav.getLink('privacy') }, ['privacy policy']), '.'
+        h(Link, { style: { textDecoration: 'underline', color: colors.accent(1.1) }, href: Nav.getLink('privacy') }, ['privacy policy']), '.'
```

Comment:
I am definitely a little concerned at how often we need to do the 1.1 colors.accent. I am going to make a note to flag that to our design team.

Reply:
True! It's definitely worth having a conversation with them about this.

Keep in mind, though, that context matters. For instance, `colors.accent(1.0)` has sufficient contrast against white, either when used as blue text on a white background (links) or white text on a blue background (primary buttons). The problem here was that the background is not white. The cookie warning uses a gray background to offset it from the main part of the page.

That said, the green color used in the header banner, `colors.primary(1.0)` does not have sufficient contrast with white, hence #2506. That one could definitely used to be darkened, although I didn't want to make that sort of global change unilaterally.

Similarly, the orange `colors.warning(1.0)` does not have nearly enough contrast against background. When I darkened it to 4.5:1 in #2538 it basically became red. So maybe that color should just go away completely and use `colors.danger()` everywhere instead.

Reply:
Great. I will add a ticket for our team to re-evaluate our base colors, and bring that to the UX team's attention. (I don't think you should worry any more than this question on it though!)